tests/data/worlds_src/__init__.py

```diff
@@ -0,0 +1,45 @@
+import glob
+import os
+import json
+
+BedrockLevels = []
+JavaVanillaLevels = []
+JavaForgeLevels = []
+
+
+java_vanilla_1_12_2 = "java/vanilla/1_12_2"
+java_vanilla_1_13 = "java/vanilla/1_13"
+
+
+def find_levels():
+    this_dir = os.path.dirname(__file__)
+    for path in glob.glob(
+        os.path.join(this_dir, "**", "world_test_data.json"), recursive=True
+    ):
+        rel_path = os.path.dirname(os.path.relpath(path, this_dir))
+        with open(path) as f:
+            test_data = json.load(f)
+        platform = test_data["world_data"]["platform"]
+        if platform == "java":
+            origin = test_data["world_data"]["origin"]
+            if origin == "vanilla":
+                JavaVanillaLevels.append(rel_path)
+            elif origin == "forge":
+                JavaForgeLevels.append(rel_path)
+            else:
+                raise Exception(f"Unknown origin {origin}")
+        elif platform == "bedrock":
+            BedrockLevels.append(rel_path)
+        else:
+            raise Exception(f"Unknown platform {platform}")
+
+
+find_levels()
+del find_levels
```

Comment:
Why does the `find_levels()` function need to be deleted after calling it? If you want to prevent it from being imported you can name it `__find_levels()` instead

Reply:
Double underscore does not stop functions from being imported from modules but I can make this change